app/logic/user.js

```diff
@@ -21,7 +21,9 @@ const Game = mongoose.model('Game');
  */
 function returnGamesWon(games, user) {
   try {
-    return games.filter(game => game.gameWinner._id.toString() === user._id.toString());
+    /* eslint max-len: 0 */
+    // disabled max-len to keep to syntax sugar
+    return games.filter(game => game.meta.gameWinner && game.meta.gameWinner.userId.toString() === user._id.toString());
   } catch (e) {
```

Comment:
Hi, you did nice work in this implementation, however, You disabled Es-lint in these line of code. It will be nice to enable the Es-lint and refactor this code to avoid complain due to linting error and also to meet the code standard. Thank you